cellengine/resources/gate.py

```diff
@@ -303,45 +464,229 @@ def create(
+        lower-right), each with a unique gid and name.
 
-class SplitGate(Gate):
-    """Basic concrete class for split gates"""
+        Args:
+            experiment_id (str): The ID of the experiment.
+            x_channel (str): The name of the x channel to which the gate applies.
+            y_channel (str): The name of the y channel to which the gate applies.
+            name (str): The name of the gate
+            x (float): The x coordinate of the center point (after the
+                channel's scale has been applied).
+            y (float): The y coordinate (after the channel's scale has been applied).
+            labels (list): Positions of the quadrant labels. A list of four length-2
+                vectors in the order: UR, UL, LL, LR. These are set automatically to
+                the plot corners.
+            skewable (bool): Whether the quadrant gate is skewable.
+            angles (list): List of the four angles of the quadrant demarcations
+            gid (str): Group ID of the gate, used for tailoring. If this is not
+                specified, then a new Group ID will be created. To create a
+                tailored gate, the gid of the global tailored gate must be specified.
+            gids (list): Group IDs of each sector, assigned to ``model.gids``.
+            locked (bool): Prevents modification of the gate via the web interface.
+            tailored_per_file (bool): Whether or not this gate is tailored per FCS file.
+            fcs_file_id (str): ID of FCS file, if tailored per file. Use ``None`` for
+                the global gate in a tailored gate group. If specified, do not
+                specify ``fcs_file``.
+            fcs_file (str): Name of FCS file, if tailored per file. An attempt
+                will be made to find the file by name. If zero or more than one
+                file exists with the name, an error will be thrown. Looking up
+                files by name is slower than using the ID, as this requires
+                additional requests to the server. If specified, do not specify
+                ``fcs_file_id``.
 
+        Returns:
+            A QuadrantGate object.
+
+        Example:
+            ```python
+            cellengine.Gate.create_quadrant_gate(experimentId, x_channel="FSC-A",
+                y_channel="FSC-W", name="my gate", x=160000, y=200000)
+            experiment.create_quadrant_gate(x_channel="FSC-A",
+                y_channel="FSC-W", name="my gate", x=160000, y=200000)
+            ```
+        """
+
+        fcs_file_id = parse_fcs_file_args(
+            experiment_id,
+            tailored_per_file,
+            fcs_file_id,
+            fcs_file,
+        )
+
+        # set labels based on axis scale
+        r = ce.APIClient().get_scaleset(experiment_id, as_dict=True)
+        scale_min = min(x["scale"]["minimum"] for x in r["scales"])
+        scale_max = max(x["scale"]["minimum"] for x in r["scales"])
```

Comment:
It'll be more accurate to find the actual scale for this gate's channels instead of averaging all of them.

Reply:
:thumbsup: